requirements/api.txt

```diff
@@ -1,4 +1,4 @@
 -r common.txt
 
 djangorestframework==3.3.2
-https://github.com/pmclanahan/django-synctool/archive/fix-dumpdata.tar.gz#egg=django-synctool==1.1.0
+https://github.com/mozmar/django-synctool/archive/multi-db-reset.tar.gz#egg=django-synctool==1.1.2
```

Comment:
I've pushed all of our changes (including your new one) to master on the mozmar fork for synctool. This will also work however. Update this to `master` if you'd like before merge, or we'll fix it for real when we get something we can use on PyPI. I'm happy either way.